front/server-config/nginx.conf

```diff
@@ -30,4 +30,8 @@ http {
     gzip                on;
 
     include             /etc/nginx/conf.d/*.conf;
```

Comment:
I think put the `include` statement at the end is a good idea.

Reply:
Also revised.